src/windows_term.rs

```diff
@@ -382,7 +382,7 @@ pub fn set_title<T: Display>(title: T) {
     }
 }
 
-#[cfg(feature = "windows-console-colors")]
+#[cfg(all(windows, feature = "windows-console-colors"))]
```

Comment:
You don't need this change since this file exists only in `windows`

Reply:
But it makes it explicit